test/LiberalParserSpec.hs

```diff
@@ -69,6 +69,16 @@ spec = do
       parseString "A ∧ B ∨ C" `shouldSatisfy` isLeft
     it "should reject \"A ∧ not B => C ∧ D\"" $
       parseString "A ∧ not B => C ∧ D" `shouldSatisfy` isLeft
```

Comment:
If the below property subsumes the above unit tests, maybe these should be replaced by some cases involving brackets (which the below property cannot capture)?